atlante/queuer/awsbatch/awsbatch.go

```diff
@@ -0,0 +1,189 @@
+package awsbatch
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/batch"
+	"github.com/go-spatial/maptoolkit/atlante"
+	cfgaws "github.com/go-spatial/maptoolkit/atlante/config/aws"
+	"github.com/go-spatial/maptoolkit/atlante/internal/env"
+	"github.com/go-spatial/maptoolkit/atlante/queuer"
+	"github.com/prometheus/common/log"
+)
+
+const (
+
+	// TYPE is the name of the provider
+	TYPE = "awsbatch"
+
+	// ConfigKeyJobDefinition is the key in the config for the job definition
+	ConfigKeyJobDefinition = "job_definition"
+
+	// ConfigKeyJobName is the key in the config
+	ConfigKeyJobName = "job_name"
+
+	// ConfigKeyJobQueue is the key in the config
+	ConfigKeyJobQueue = "job_queue"
+
+	// ConfigKeyJobParams is the key in the config
+	ConfigKeyJobParams = "job_parameters"
+
+	// ConfigKeyJobObjectKey is the config key for the job object that will be used
+	// added to the params
+	ConfigKeyJobObjectKey = "job_object_key"
+
+	// DefaultJobObjectKey is the default that is used by if job_object_key
+	// is not set.
+	DefaultJobObjectKey = "job-data"
+)
+
+func init() {
+	queuer.Register(TYPE, initFunc, nil)
+}
+
+type Provider struct {
+	Definition string
+	Name       string
+	Queue      string
+	Params     map[string]string
+	ObjectKey  string
+	C          *batch.Batch
```

Comment:
What does "C" stand for? "client"? If so, can we name it "Client"? I find "C" difficult to understand as a public property on a struct.